cmake/KokkosCore_config.h.in

```diff
@@ -72,6 +72,10 @@
 #cmakedefine KOKKOS_ENABLE_ONEDPL
 #cmakedefine KOKKOS_ENABLE_ROCTHRUST
 
+/* Embedded dependencies */
+#define KOKKOS_IMPL_DESUL_VERSION "@KOKKOS_DESUL_VERSION@"
+#define KOKKOS_IMPL_MDSPAN_VERSION "@KOKKOS_MDSPAN_VERSION@"
+
```

Comment:
What if a user disables `desul` ? Do we still want the `KOKKOS_IMPL_DESUL_VERSION` to be defined ?